bitrise.yml

```diff
@@ -149,29 +233,17 @@ workflows:
 
             echo "-- unzip -- "
             exec unzip "${BITRISE_ARTIFACT_PATHS}"
-            echo "show dir"
-            ls
-    - script@1.1:
-        run_if: '{{getenv "RUN_UI_TESTS" | eq "Run_UI_Tests"}}'
-        title: Test without Building
-        inputs:
-        - content: |
-            #!/usr/bin/env bash
-            set -ex
-
-            ls ./Users/vagrant/git/Tests/
-            cp -r ./Users/vagrant/git/Tests/* .
-            mv ./Users/vagrant/git/DerivedData .
-            ls -la
-            echo "-- test-without-building --"
-            xcodebuild -resultBundlePath "xcodebuild.xcresult" -derivedDataPath "/Users/vagrant/git/DerivedData" -destination "platform=iOS Simulator,name=iPhone 14,OS=latest" test-without-building -xctestrun "/Users/vagrant/git/DerivedData/Build/Products/Fennec_Enterprise_XCUITests_Smoketest2_iphonesimulator16.1-x86_64.xctestrun"
-    - custom-test-results-export@0:
+    - xcode-test-without-building@0:
```

Comment:
Instead of using a script ste to test without building, we are using now a pre-built step given by Bitrise.